
class PyTSSError(Exception): pass
class ErrorGeneratingPrime(PyTSSError): pass